platforms/chibios/chibios_config.h

```diff
@@ -109,3 +112,12 @@
 #    define PAL_OUTPUT_TYPE_PUSHPULL PAL_HT32_MODE_DIR
 #    define PAL_OUTPUT_SPEED_HIGHEST 0
 #endif
+
+#if defined(MCU_MIMXRT1062)
+#    include "clock_config.h"
+#    define CPU_CLOCK BOARD_BOOTCLOCKRUN_CORE_CLOCK
+#endif
```

Comment:
A bit pedantic, but can we move this under the `MCU_KINETIS` block so the teensy stuff is next to each other

Reply:
No problem, done.